### PR TITLE
fix(services): Stop escaping characters when using links with special characters (SQSERVICES-1342)

### PR DESCRIPTION
**Pull request by thisisamir98**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/SQSERVICES-1342" title="SQSERVICES-1342" target="_blank">BugSQSERVICES-1342</a>  [Web] Fix escaping characters when using links with special characters
  </td></table>
  <br />

### CODE REVIEW

File: src/script/util/messageRenderer.ts [hunk 1]
```diff
@@ -175,9 +175,9 @@ export const renderMessage = (message: string, selfId: QualifiedId | null, menti
     if (!isWireDeepLink && !['autolink', 'linkify'].includes(link.markup)) {
       const title = link.attrGet('title');
       if (title) {
-        link.attrSet('title', cleanString(title));
+        link.attrSet('title', title);
```
Comment (atomrc) [inline src/script/util/messageRenderer.ts:178]: We are sure there is no risk for the title, right?

**Review by atomrc — APPROVED** [review]

File: src/script/util/messageRenderer.ts (hunk 1 above)
Comment (comawill) [reply src/script/util/messageRenderer.ts:178]: @atomrc yes. attributes set by attrSet are esacaped by markdown-it, code flow from renderToken to escapeHTML:
1. https://github.com/markdown-it/markdown-it/blob/e07a9ddeee192ad099ed1dd7e6d1960fd5aa8d05/dist/markdown-it.js#L3210
2. https://github.com/markdown-it/markdown-it/blob/e07a9ddeee192ad099ed1dd7e6d1960fd5aa8d05/dist/markdown-it.js#L3191
3.  https://github.com/markdown-it/markdown-it/blob/e07a9ddeee192ad099ed1dd7e6d1960fd5aa8d05/dist/markdown-it.js#L2748

@thisisamir98 tests must be adjusted, as the output has changed in some cases. Please also include tests covering the following inputs:
* `[sometext](https://some.domain"><script>alert("oops")</script>)`
* `[sometext](https://some.domain?param1=a&param2=b)`
* `[sometext](https://some.domain "this is a title")`
* `[sometext](https://some.domain "<this> &' >that<")`
* `[sometext](https://some.domain '"><script>alert("oops")</script>')`

**Review by comawill — CHANGES_REQUESTED** [review]
Please add tests (see my last comment)

File: src/script/util/messageRenderer.ts (hunk 1 above)
Comment (thisisamir98) [reply src/script/util/messageRenderer.ts:178]: https://github.com/wireapp/wire-webapp/pull/12795/commits/9aa7251ab7e09e001dbd2f8ea8694e608a21d268

**Review by comawill — APPROVED** [review]

**Pull request merged**